raiden-dapp/src/services/raiden-service.ts

```diff
@@ -36,6 +36,7 @@ export default class RaidenService {
     stateBackup?: string,
     subkey?: true
   ): Promise<Raiden> {
+    const revealTimeout = process.env.VUE_APP_REVEAL_TIMEOUT === undefined ? undefined : parseInt(process.env.VUE_APP_REVEAL_TIMEOUT);
```

Comment:
This is ugly! Is there a nicer way to do this?

Reply:
```suggestion
    const revealTimeout = +process.env.VUE_APP_REVEAL_TIMEOUT ? +process.env.VUE_APP_REVEAL_TIMEOUT : undefined;
```
The unary operator parses the number, and resolves to `NaN` if it can't (which is falsy). You can use an intermediary variable to store the parsing result, but I think it's not worth it

Reply:
With your proposal I get 
```
Object is possibly 'undefined'.
```

Reply:
I inlined the old solution